src/backend/access/transam/csn_log.c

```diff
@@ -215,11 +216,12 @@ CSNLogGetCSNByXid(TransactionId xid)
 	int			pageno = TransactionIdToPage(xid, current_region);
 	int			entryno = TransactionIdToPgIndex(xid);
 	int			slotno;
+	XLogRecPtr  min_lsn = InvalidXLogRecPtr;
 	XidCSN *ptr;
 	XidCSN	xid_csn;
 
 	/* lock is acquired by SimpleLruReadPage_ReadOnly */
-	slotno = SimpleLruReadPage_ReadOnly(CsnlogCtl, pageno, xid);
+	slotno = SimpleLruReadPage_ReadOnly(CsnlogCtl, pageno, xid, min_lsn);
```

Comment:
@ctring  This is the slot where the min_lsn needs to be populated from the VisibilityCheck.